### ISSUE TITLE
vdom error

### ISSUE THREAD

**Issue by chancyk** [yysun/apprun-f7#7]
I was experimenting when adding another component from Framework7 pulled from their kitchen sink examples. The example include inline style attributes, which causes the vdom to throw an error:

```
Uncaught TypeError: Failed to set an indexed property on 'CSSStyleDeclaration': Indexed property setter is not supported.
    at d (vdom-my.ts:173)
    at a (vdom-my.ts:144)
    at vdom-my.ts:146
    at Array.forEach (<anonymous>)
    at a (vdom-my.ts:146)
    at vdom-my.ts:146
    at Array.forEach (<anonymous>)
    at a (vdom-my.ts:146)
    at f (vdom-my.ts:117)
    at h (vdom-my.ts:47)
```

Any idea what might be going on here?

**Comment by yysun** [comment]
Can you please show me what is the style you were trying to add? Or which component?

**Comment by chancyk** [comment]
I added the following:
```typescript
export const ExpandableCard = (props, children) => (
  <div class="card card-expandable">
    <div class="card-content">
      <div class="bg-color-red" style="height: 300px">
        <div class="card-header text-color-white display-block">Framework7<br />
          <small style="opacity: 0.7">Build Mobile Apps</small>
        </div>
        <a href="#" class="link card-close card-opened-fade-in color-white"
          style="position: absolute; right: 15px; top: 15px">
          <i class="icon f7-icons">xmark_circle_fill</i>
        </a>
      </div>
      <div class="card-content-padding">{children}</div>
    </div>
  </div>
);
```
The first failure was on `height: 300px` and it progressed to the next style `opacity: 0.7` when I removed height. I believe [this](https://github.com/yysun/apprun/blob/6bcfc1275555c6da39ef4eb6f9f67601292e5c38/src/vdom-my.ts#L184) is the line that fails when it tries to index `element.style`.

I think it's possibly intending to index by 'height' but it's receiving an ordinal index from `for (const s in value)` since `value` is a string.

**Comment by chancyk** [comment]
Your commit a6b64382 seems to have fixed my issue! :thinking:

**Comment by yysun** [comment]
Interesting. The commit is just an update to the demo. No change to AppRun
itself.

**Comment by chancyk** [comment]
I rolled back to 2.23.1 and was able to reproduce the issue, and it was resolved some time before 2.24.0. My fault, I was testing with the old version. I really appreciate the quick response though, and thanks for updating the dependency versions! 😄
